main_test.go

```diff
@@ -217,4 +217,56 @@ func TestDownload_Chunks(t *testing.T) {
 	}
 }
 
-// TODO: add tests for getDownloadSize (success with Content-Length, success with Content-Range, failure)
+func TestGetDownloadSize_ContentLength(t *testing.T) {
+	s := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			fmt.Fprint(w, "Test")
+		},
+	))
+	defer s.Close()
+
+	d := DefaultDownloader()
+	d.client = &http.Client{Timeout: d.TimeoutPerChunk}
```

Comment:
You're completely right! But that's a bug. I mean… I expect `DefaultDownloader()` to give us a fully working downloader. Would you ming fixing this bug?

I think that moving this line from `Downloader.DownloadWithContext(…)` to `DefaultDownloader()` might do the trick:

https://github.com/cuducos/chunk/blob/a27be85ef09709d8ac15e7804eedf09dbd879cd0/main.go#L211

Otherwise, let me know and I add an issue to handle that later : )

Reply:
Yeah, absolutely! Will fix that